CHANGELOG.md

```diff
@@ -6,14 +6,92 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/).
 
 ## Upcoming release
 
-### Added
-
-* Support for Bazel 2.0.0.
-
-### Changed
+nothing yet
+
+### [0.12.0] - 2020-03-16
+
+## Highlights
+
+* Various improvements to Windows support.
+
+* Support for Bazel 2.0.0
+
+* Minimum supported Bazel version is now 0.29, but GHC bindist is known to
+  [fail on macOS](https://app.circleci.com/jobs/github/tweag/rules_haskell/8684)
+  with Bazel 0.29. Please use a more recent version for macOS.
+
+## Added
+
+* `haskell_repl` now has a `hie_bios` output group
+  See [#1263](https://github.com/tweag/rules_haskell/pull/1263)
+* Added support for [hrepl](https://github.com/google/hrepl)
+  (a standalone binary that runs REPLS for Bazel Haskell targets).
+  See [#1210](https://github.com/tweag/rules_haskell/pull/1210).
+* `haskell_cabal_library`, `haskell_cabal_binary`, and `stack_snapshot`
+  now have a `verbose` argument, to allow suppressing their output.
+  See [#1208](https://github.com/tweag/rules_haskell/pull/1208).
+* `haskell_cabal_library` and `stack_snapshot` now have a
```

Comment:
I think this and https://github.com/tweag/rules_haskell/pull/1279/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR40 could be merged into one entry as #1200 is just about making #1102 optional. Maybe something like

```markdown
 * `haskell_cabal_library` and `stack_snapshot` now build the haddock documentation if the parameter `haddock` is set to `True` (the default).
  See [#1102](https://github.com/tweag/rules_haskell/pull/1102) and  [#1200](https://github.com/tweag/rules_haskell/pull/1200)

Reply:
Implemented this suggestion in 1191ebf